GLOSSARY.md

```diff
@@ -0,0 +1,31 @@
+<dl>
+  <dt>GPR</dt>
+  <dd>gaussian process regression</dd>
+
+  <dt>SVGP</dt>
+  <dd>stochastic variational inference for Gaussian process models</dd>
+
+  <dt>[] shape constructions</dt>
+  <dd>the way of describing tensor shape in docstrings and comments. Example: [..., N, D, D], this is tensor with unknown number of leading dimensions and last two dimensions are equal</dd>
+
+  <dt>X</dt>
+  <dd>and its variations like Xnew in GPflow code mean input data (numpy or tensorflow)</dd>
+
+  <dt>Y</dt>
+  <dd>throughout is used to denote observed data, potentially with multiple output dimensions</dd>
```

Comment:
```suggestion
  <dd>refers to observed output values, potentially with multiple output dimensions; always of rank 2, e.g. shape [N,
 P], even when P=1</dd>
```
Would it be useful to point towards the notebooks here (Coregionalization still has "P=1", the outputs are distinguished by one of the columns in the input points)?